mne/source_space.py

```diff
@@ -1426,12 +1432,18 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
     """
     subjects_dir = get_subjects_dir(subjects_dir)
 
+    if fname is not None or overwrite is not None:
+        warn("Parameters 'fname' and 'overwrite' are deprecated and will be "
+             "removed in version 0.15. The source spaces will not be saved. "
+             "Use mne.write_source_spaces instead.")
     if bem is not None and surface is not None:
         raise ValueError('Only one of "bem" and "surface" should be '
                          'specified')
     if mri is not None:
         if not op.isfile(mri):
-            raise IOError('mri file "%s" not found' % mri)
+            mri = op.join(subjects_dir, subject, 'mri', mri)
```

Comment:
...and actually we could/maybe should save and load `subject_his_id` like we do for surface-based source spaces.